CRITICAL - 2021-04-29 12:43:33 --> Invalid file: halo_view.php
#0 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\View\View.php(203): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('halo_view.php')
#1 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('halo_view', Array, true)
#2 C:\xampp\htdocs\santrikoding\app\Controllers\Halo.php(11): view('halo_view', Array)
#3 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Halo->index()
#4 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Halo))
#5 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\santrikoding\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\santrikoding\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-04-29 12:45:10 --> Cannot declare class App\Controllers\Halo, because the name is already in use
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
ERROR - 2021-04-29 13:24:48 --> Error connecting to the database: Access denied for user '****'@'localhost' (using password: YES)
